dts/msm8916/msm8916-mtp.dts

```diff
@@ -39,4 +39,13 @@
 		compatible = "asus,z010d", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:none";
 	};
+
+	lenovo-phab {
+		qcom,msm-id = <206 0>;
+		qcom,board-id = <8 0>;
```

Comment:
Don't think you need to override the board to the same value as it was?

Reply:
didn't understand your question

Reply:
it is, downstream: https://github.com/k1gen/android_kernel_lenovo_phab/blob/e3cfd20e601016cc44f7b233248a64a7290135cb/arch/arm64/boot/dts/qcom/msm8916-mtp.dts#L21